widgets/GCSMove-action.json

```diff
@@ -128,6 +128,16 @@
           }
         }
       ]
+    },
+    {
+      "label" : "Advanced",
+      "properties" : [
+        {
+          "name": "readTimeout",
+          "widget-type": "textbox",
```

Comment:
use widget type number, for example: https://github.com/data-integrations/google-cloud/blob/b4c7d043eb84b6611c2d70554233d35cab32ff70/widgets/GCSFile-batchsource.json#L122

This comment applies to whole PR.

Reply:
Addressed